lib/functions.php

```diff
@@ -21,6 +21,7 @@
 define('DOCDIR',sprintf('%s/',realpath(LIBDIR.'../doc/')));
 define('HOOKSDIR',sprintf('%s/',realpath(LIBDIR.'../hooks/')));
 define('JSDIR','js/');
+define('SESSION_CIPHER','aes256-gcm');
```

Comment:
This is `aes-256-gcm` on my side. I think you're just skipping the OpenSSL part since `in_array(SESSION_CIPHER, openssl_get_cipher_methods())` is falsy?

Reply:
That's true for me too, on Debian PHP 8.2 !
cc @tachtler

Reply:
Okay, fixed by https://github.com/leenooks/phpLDAPadmin/commit/ef8d0ce94cbb5f17f5a0b25b9bc571e8eb8c5419